app/models/import_batch.rb

```diff
@@ -30,6 +30,11 @@ class ImportBatch < MappingsBatch
     validator: HostInWhitelistValidator,
     message: I18n.t('mappings.bulk.new_url_must_be_on_whitelist')
   }
+  validates :archive_urls, each_in_collection: {
+    validator: LengthValidator,
+    maximum: (64.kilobytes - 1),
+    message: I18n.t('mappings.new_url_too_long')
```

Comment:
You're right, it might be confusing to users!

I believe in this case, the user has _just_ finished uploading their CSV of "Old URL,New URL", so there's no distinct field for an 'Archive URL' at this point, and 'New URL' still refers to the column name.

Which is the last in your list of possible solutions - take the name of the field - so I assume this is probably our best option right now?